content/en/docs/reference/command-line-tools-reference/feature-gates/index.md

```diff
@@ -0,0 +1,527 @@
+| `ServerSideFieldValidation` | `false` | Alpha | 1.23 | 1.24 |
+| `ServerSideFieldValidation` | `true` | Beta | 1.25 | 1.26 |
+| `ServerSideFieldValidation` | `true` | GA | 1.27 | - |
+| `ServiceIPStaticSubrange` | `false` | Alpha | 1.24 | 1.24 |
+| `ServiceIPStaticSubrange` | `true` | Beta | 1.25 | 1.25 |
+| `ServiceIPStaticSubrange` | `true` | GA | 1.26 | - |
+| `ServiceInternalTrafficPolicy` | `false` | Alpha | 1.21 | 1.21 |
+| `ServiceInternalTrafficPolicy` | `true` | Beta | 1.22 | 1.25 |
+| `ServiceInternalTrafficPolicy` | `true` | GA | 1.26 | - |
+| `TopologyManager` | `false` | Alpha | 1.16 | 1.17 |
+| `TopologyManager` | `true` | Beta | 1.18 | 1.26 |
+| `TopologyManager` | `true` | GA | 1.27 | - |
+| `WatchBookmark` | `false` | Alpha | 1.15 | 1.15 |
+| `WatchBookmark` | `true` | Beta | 1.16 | 1.16 |
+| `WatchBookmark` | `true` | GA | 1.17 | - |
+| `WindowsHostProcessContainers` | `false` | Alpha | 1.22 | 1.22 |
+| `WindowsHostProcessContainers` | `true` | Beta | 1.23 | 1.25 |
+| `WindowsHostProcessContainers` | `true` | GA | 1.26 | - |
+{{< /table >}}
+
+## Using a feature
+
+### Feature stages
+
+A feature can be in *Alpha*, *Beta* or *GA* stage.
+An *Alpha* feature means:
+
+* Disabled by default.
+* Might be buggy. Enabling the feature may expose bugs.
+* Support for feature may be dropped at any time without notice.
+* The API may change in incompatible ways in a later software release without notice.
+* Recommended for use only in short-lived testing clusters, due to increased
+  risk of bugs and lack of long-term support.
+
+A *Beta* feature means:
+
+* Enabled by default.
+* The feature is well tested. Enabling the feature is considered safe.
+* Support for the overall feature will not be dropped, though details may change.
+* The schema and/or semantics of objects may change in incompatible ways in a
+  subsequent beta or stable release. When this happens, we will provide instructions
+  for migrating to the next version. This may require deleting, editing, and
+  re-creating API objects. The editing process may require some thought.
+  This may require downtime for applications that rely on the feature.
+* Recommended for only non-business-critical uses because of potential for
+  incompatible changes in subsequent releases. If you have multiple clusters
+  that can be upgraded independently, you may be able to relax this restriction.
+
+{{< note >}}
+Please do try *Beta* features and give feedback on them!
+After they exit beta, it may not be practical for us to make more changes.
+{{< /note >}}
+
+A *General Availability* (GA) feature is also referred to as a *stable* feature. It means:
+
+* The feature is always enabled; you cannot disable it.
+* The corresponding feature gate is no longer needed.
+* Stable versions of features will appear in released software for many subsequent versions.
+
+## List of feature gates {#feature-gates}
+
+Each feature gate is designed for enabling/disabling a specific feature:
+
```

Comment:
If this PR merges, I can imagine later updating to a custom page layout. The custom layout could user a localizable [partial](https://gohugo.io/templates/partials/) so that you don't have to list all the feature gates here.

If we do that, each localization would instantly benefit from the correct list of feature gates (albeit some might be in English). We could even use a marker to highlight which are in the “wrong” language.